packages/yoga/src/IconWrapper/IconWrapper.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { oneOf, elementType, string } from 'prop-types';
+import { withTheme } from 'styled-components';
+
+const IconWrapper = ({
+  as: Component,
+  width,
+  height,
+  fill,
+  stroke,
+  theme,
+  ...props
+}) => (
+  <Component
+    width={theme.yoga.spacing[width] || width}
+    height={theme.yoga.spacing[height] || height}
```

Comment:
`spacing` is an array with [some aliases](https://github.com/Gympass/yoga/blob/master/packages/tokens/src/global/spacing.js) mapping its indexes.

```javascript
theme.yoga.spacing[5] === theme.yoga.spacing.small // true
```

So, if you want `width`, for example, to be `5px` you just pass it as `"5px"`. If you pass it as `5` or `small`, it would set the SVG width as 16px (current `spacing[5]`)

This prop should be `string` or `number`. Right?

Reply:
Yeah, it's just that plain numbers are also valid for these attributes, we are "reserving" `0` to `11`.

But if that's clear, I see no problem!